WordPress/Sniffs/WP/AlternativeFunctionsSniff.php

```diff
@@ -92,6 +92,24 @@ public function getGroups() {
 				),
 			),
 
+			'rand_seeding' => array(
+				'type'      => 'warning',
+				'message'   => '%s() is discouraged. Rand seeding is not necessary when using the wp_rand() function (as you should).',
+				'functions' => array(
+					'srand',
+					'mt_srand',
+				),
+			),
+
+			'rand' => array(
+				'type'      => 'warning',
+				'message'   => '%s() is discouraged. Use the far less predictable wp_rand() instead.',
+				'functions' => array(
+					'rand',
+					'mt_rand'
```

Comment:
Missing comma here is causing the Travis build failure.

https://travis-ci.org/WordPress-Coding-Standards/WordPress-Coding-Standards/jobs/325103528#L785-L794

Reply:
Thanks, Fixed now.